README.md

```diff
@@ -1 +1,170 @@
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [portfolio Project] <a name="about-project"></a>
+
+**[portfolio Project]** is a simple HTML, CSS and Javascript webpage design from module 1 of Microverse.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.org/">HTML</a></li>
+    <li><a href="w3school.com/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Add html webpage]**
+- **[Add readme.md files]**
+- **[Add MIT license file]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="https://https://peteronyam.github.io/conference/"></a>
+
+- [Live Demo Link](https://https://peteronyam.github.io/conference/)
+- - [Live Intro Video](https://www.loom.com/share/124738486bc4459fae419cf521be716e)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+**A web Browser
+A text editor
+Basic knowledge of HTML, CSS and JAVASCRIPT
+Git (optional)**
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  **git clone https://git@github.com:peteronyam/conference.git**
+
+### Install
+
+Install this project with:
+
+  **npm install**
+
+### Usage
+
+To run the project, execute the following command:
+
+**npm start**
+
+### Run tests
+
+To run tests, run the following command:
+
+  **npm test**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@peteronyam](https://github.com/peteronyam)
+- Twitter: [@jiyapete](https://twitter.com/jiyapete)
+- LinkedIn: [peteronyam](https://linkedin.com/in/peteronyam)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Updating design]**
+- [ ] **[updating styling]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```

Comment:
- [ ] ReadMe content: In reading from your project requirements, it says we should give credit to the author of the design. Therefore, I would advise you to acknowledge or give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).